scripts/simplify_network.py

```diff
@@ -570,14 +728,14 @@ def consense(x):
         )
         busmaps.append(stub_map)
 
-    if cluster_config.get("to_substations", False):
+    if cluster_config.get("to_substations", 0.0):
```

Comment:
I think this False was ok, when false, we don't want to use the to_substations aggregations

Reply:
Fixed, thanks!